set.mm

```diff
@@ -437541,10 +437541,10 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
  latexdef "/_f" as "\mathrm{/}_f";
 htmldef "_O" as '_O';
  althtmldef "_O" as "&#x39F;";
- latexdef "_O" as "\Omicron";
+ latexdef "_O" as "O";
```

Comment:
Why ? Is this for asymptotic comparisons, e.g. f(x) = \Omicron (x) ?

Reply:
Yes, in set.mm it's described as <code>the class of the "big-O" function</code>, but the reason I replaced it is because <code>\\Omicron</code> generates the error <code>undefined control sequence. \end{align}</code> in latex (see point 3 of the pull request description).

Reply:
Ok. Then "\mathrm{O}".